### PR TITLE
Implemented #200 dropdown update

### PR DESCRIPTION
**Pull request by samiib**
-When worker nodes are added/removed, dropdown is updated now. :)

### CODE REVIEW

File: terraform/appflask.py [hunk 1]
```diff
@@ -828,13 +822,27 @@ def calico_nodes_view():
...
                                 ipv6_cluster_subnet=ipv6_cluster_subnet,
                                 calico_nodes=json.dumps(calico_nodes, indent=4),
-                                hostnames = hostnames,
+                                hostnames = get_hostnames(calico_nodes),
                                 manage_cluster=manage_cluster,
                                 fabric_type=fabric_type)
 
 
+def stream_calico_nodes_update(calico_nodes):
+    return turbo.stream([
```
Comment (camrossi) [inline terraform/appflask.py:831]: I think we should do if turbo can stream then stream

File: terraform/appflask.py [hunk 2]
```diff
@@ -761,9 +761,8 @@ def calico_nodes_view():
             calico_nodes.append({"hostname": hostname, "ip": ip,
                                 "ipv6": ipv6, "natip": natip, "rack_id": rack_id})
             if turbo.can_stream():
```
Comment (camrossi) [inline terraform/appflask.py:763]: You have it here (I am referring to the next comment) but maybe is better to put it all in one place?

File: terraform/appflask.py (hunk 1 above)
Comment (samiib) [reply terraform/appflask.py:831]: The question is what would happen if turbo can't stream.  Does the whole thing just crash?

Comment (camrossi) [reply terraform/appflask.py:831]: Pretty much but at least we know something is wrong and we don't offer back stale/broken data. I prefer to fail hard then to silently fail. 

Yes Ideally we should handle this in a smarter way :D

Comment (samiib) [reply terraform/appflask.py:831]: Okay I'll make the change. 👍

File: terraform/appflask.py (hunk 2 above)
Comment (samiib) [reply terraform/appflask.py:763]: You mean inside right

**Pull request merged**